templates/config.yml

```diff
@@ -93,6 +93,9 @@ blockchain_adapters:
 # You can also set a testnet insight url to use:
 #   insight_test_url: "https://test.localhost/api"
 
+# Websocket Insight Url
+insight_websocket_url: "wss://insight.mycelium.com"
```

Comment:
Perhaps it's not such a great idea to add this to config as a default option. Let's use Bitpay's one. We don't want all people using straight to also abuse our Insight (same goes for `spec/.straight/config.yml`)